src/FluentlyHttpClient/Utils/GqlQuery.cs

```diff
@@ -0,0 +1,20 @@
+﻿// ReSharper disable once CheckNamespace
+namespace FluentlyHttpClient
+{
+	/// <summary>
+	/// Request object for GraphQL requests
+	/// </summary>
+	public class GqlQuery
+	{
+
+		/// <summary>
+		/// GraphQL query string
```

Comment:
for consistency - **Gets or sets GraphQL query.** (same for the others)
 - remove query string
 - use period